example.py

```diff
@@ -169,7 +221,3 @@ def editor_multi_agent_example():
         for _ in range(1000):
             states = env.tick()
 
-
-if __name__ == "__main__":
```

Comment:
I'm not against removing this, but I just want to make sure it was intentional.

Reply:
Twas